src/pages/Landing/index.js

```diff
@@ -1,29 +1,39 @@
-import React from 'react';
-import LandingPageWrapper from '../../components/LandingPageWrapper';
+import React, { Component } from 'react';
+import ArticleContainer from '../../components/Articles/ArticleContainer';
 import LandingImage from '../../assets/images/landing_image.jpg';
+import ARTICLE_DATA from '../../data/sample_articles.js';
 
 import './Landing.scss';
 
-
-const Landing = () => (
-    <div>
-      <div className="page-banner-image">
-         <img className='LandingImage' src={LandingImage} alt="Landing image" />
-      </div>
-      <div className="small-screen">
-        <div>
+class Landing extends Component {
```

Comment:
Hey @felixkiryowa, this should be a stateless component

Reply:
This component should connect to redux, therefore it should be a smart component